en/FlyView/FlyView.md

```diff
@@ -182,10 +182,10 @@ Return to the home position at any time while flying:
 1. Press the **RTL** button in the *Fly Tools*.
 1. Confirm RTL using the slider.
 
-![land](../../assets/fly/land.jpg)
+![land](../../assets/fly/rtl.jpg)
 
 > **Note** The vehicle may also land at the home position, depending on its type and configuration.
-
+** The above Note is not clear, after RTL, the vehicle is suppose to land at home position only**
```

Comment:
Thanks, you're right and you're wrong. The note is unclear, but also correct. Depending on configuration a vehicle may not fly to the home position (it could go to a mission landing or to a rally point. Depending on configuration it may not land - it could just hover or loiter. I will reword.

Reply:
PS Easiest way to mark this kind of thing up is just to put a space at the end of the problem line and then add a comment in review.